doc/api/addons.md

```diff
@@ -213,6 +213,24 @@ Addon developers are recommended to use to keep compatibility between past and
 future releases of V8 and Node.js. See the `nan` [examples][] for an
 illustration of how it can be used.
 
+
+## N-API
+
+N-API is a newer API for building native Addons.  It is independent from
```

Comment:
I don't know if we standardized anywhere, but you have two spaces after some sentences, and one period after others.